src/main/java/org/jabref/logic/util/strings/UnicodeLigaturesMap.java

```diff
@@ -0,0 +1,67 @@
+package org.jabref.logic.util.strings;
+
+import java.util.HashMap;
+
+public class UnicodeLigaturesMap extends HashMap<String, String> {
```

Comment:
Why not using a BiMap from Guava? As far as I know we already do that for the html/latex mapping as well.
I would be in favor of this - as a way to convert back

Reply:
I had a quick look at this. `BibMap` is an interface and its implementations (e.g. `HashBiMap`) don't have constructors, so I cannot extend them.  So, I would have to implement all map methods anew.

Although it would be nice, let us wait with something like that until we have an actual use case. Right now, reversing a standard Java Map is much less code than hooking into the Guava collections.